docs/guide/cell-references.md

```diff
@@ -160,7 +160,38 @@ copied between different places.
   </tbody>
 </table>
 
-## Circular reference
+## Range references
+
+In HyperFormula, a range is a group of two or more adjacent cells.
```

Comment:
>  Range `<Addr1>:<Addr2>` is a reference to the smallest 3-dimensional cube of cells that include both `Addr1` and `Addr2` including the cells on sheets positioned between them.

IMO this sentence describes it pretty well. Although it might be too technical for some users. @kirszenbaum I leave it up to you if you want to use this sentence or not.
 
 From: https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part4-formula/OpenDocument-v1.3-os-part4-formula.html#__RefHeading__1018032_715980110

Reply:
@sequba I went for a simpler, "end-user" option because I wasn't so sure about the ODF standard's definition, particularly about the **3-dimensional cube** metaphor:
- It's a tautology (a cube that has three dimensions by default)
- A cube has 12 edges of identical length (that's not the case with ranges, right?). I think they meant **cuboid** here
- Looking at [this](https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part4-formula/OpenDocument-v1.3-os-part4-formula.html#Reference), all we want to say by mentioning "3 dimensions" is that ranges support cross-sheet referencing, am I right?

To make that definition consistent with the [4.8 ref definitions](https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part4-formula/OpenDocument-v1.3-os-part4-formula.html#Reference), we'd need to reword it like this:

> Range `<Cell position 1>:<Cell position 2>` is a reference to the smallest possible cell cuboid that includes `<Cell position 1>`, `<Cell position 2>`, and all cells on all sheets positioned between `<Cell position 1>` and `<Cell position 2>`.

But IMO the above only makes sense when you're actually familiar with the [4.8 Reference](https://docs.oasis-open.org/office/OpenDocument/v1.3/os/part4-formula/OpenDocument-v1.3-os-part4-formula.html#Reference) terminology: **cell position**, **cell strip**, **cell rectangle**, **cell cuboid**, and **reference**. And we're not using that terminology anywhere else in the HF docs. I believe the definition should be more-or-less self-explanatory for people not familiar with the ODF standard.

I'd also differentiate between a **range** of cells and a **range reference** (a reference to a range of cells).

TL;DR
How about something like this:

> Range `<Cell address 1>:<Cell address 2>` is the smallest possible group of adjacent cells that includes:
> - The cell at `<Cell address 1>`
> - The cell at `<Cell address 2>`
> - All cells on all sheets between `<Cell address 1>` and `<Cell address 2>`
>
> Tip: In HyperFormula, a single cell is not treated as a range.

WDYT?

Reply:
Ok, the word 'cube' is inaccurate indeed. It was only a suggestion. What you propose is better.

Another idea that I have is to define a 2D range and than say, that it's also possible to span it across multiple sheets, so something like this:

> Range `<Cell address 1>:<Cell address 2>` is the smallest rectangular collection of cells including both `<Cell address 1>` and `<Cell address 2>`.
>
> Note: Ranges can also span multiple sheets. `<SheetA>!<Cell address 1>:<SheetB>!<Cell address 2>` includes the cells from `SheetA`, `SheetB` and all the sheets between them (ID-wise).

 As I said, I'm leaving the decision to you because you are in a better position to know what wording is more clear for ourusers.